### ISSUE TITLE
[BUG] Reader API: GoogleLogin auth token in POSTDATA query string is ignored

### ISSUE THREAD

**Issue by lwindolf** [FreshRSS/FreshRSS#5287]
**Describe the bug**
When a Google Reader API client provide the GoogleLogin auth token in a POSTDATA provided query string the auth token is not processed and a HTTP 401 is returned. This happens with https://github.com/lwindolf/liferea/issues/1198.

When the same auth token is provided via `Authorization` header everything works.

**To Reproduce**
1. Run Liferea 1.4.4 as client
2. Add FreshRSS 1.21.0 instance as source
3. Try to fetch items

**Expected behavior**
Auth token is also accepted from POSTDATA

**Comment by Alkarex** [comment]
When an API request fails, you should have an entry in your `./FreshRSS/data/users/_/log_api.txt`
Could you please share it here (anonymised)?

**Comment by Alkarex** [comment]
Tokens are always passed via the `Authorization` HTTP header, not in a POST payload.
Are there any clients or servers (using the GReader API) not doing that (besides Liferea)?

* https://feedhq.readthedocs.io/en/latest/api/terminology.html#authentication
* https://github.com/theoldreader/api#using-a-token
* https://github.com/bazqux/bazqux-api#token
* Etc.

**Comment by lwindolf** [comment]
@Alkarex For testing I did use a hosted setup so I sadly cannot provide the debug log :-(

As for other readers doing it via POST payload too I do not know. The original documentation (e.g. https://gist.github.com/alexwilson/e24332ed76d561ca1518f21146e5b4e2) was just a mess, but it indicates for the EDIT API that the token parameter is in the POST payload. I did not find documentation that this extends to the Streams API, I was just assuming it and it works with at least Miniflux.

In the end for the Liferea use case it is not important anymore as I did switch the implementation to using the header. I just wanted to report this to let you know about this possibility. So feel free to close the bug as a corner case or undocumented behaviour.
